src/assessor_tools.yml

```
@@ -294,6 +316,19 @@
         powershell: yes
         unarchive_extra_opts:
           - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/skelsec/pypykatz/tarball/master
+        install_dir: /tools/pypykatz
+        pip_packages:
+          - aiowinreg
+          - minidump
+          - minikerberos
+          - msldap
+          - pypykatz
+          - winacl
```

All of the requirements listed here are found in the `setup.py` `install_requires` of the source. As configured this also does not install the GitHub version of the tool.

```suggestion
          - .
```

---

I wasn't familiar with the proper way to install requirements when a `requirement.txt` file wasn't present. Now I know and will do so in the future.